docs/content/guides/dagster/managing-ml.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Managing machine learning models with Dagster | Dagster Docs
+description: This guide illustrates how to use Dagster to automate and manage your machine learning pipeline
+---
+
+# Managing machine learning models with Dagster
+
+## Prerequisites
+
+Before proceeding, we recommend reading the following guides:
+
+- [Automating your data pipelines](docs/content/guides/dagster/automating-pipelines)
+- [Building machine learning pipelines with Dagster](docs/content/guides/dagster/ml-pipeline)
+
+--- 
+
+## Overview 
+
+This guide will review ways to manage and maintain your machine learning (ML) model in Dagster.
+
+Machine learning models are highly dependent on data at a point in time and must be managed to ensure they produce the same results as when you were in the development phase. In this guide, you'll learn how to:
+
+- Automate training of your model when new data is available or when you want to use your model for predictions
+- Integrate metadata about your model into the Dagster UI to display info about your model's performance
+
+---
+
+## Machine learning operations
+
+You might have thought about your data sources, feature sets, and the best model for your use case. Inevitably, you start thinking about how to make this process sustainable and operational and deploy it to production. You want to make the machine learning pipeline self-sufficient and have confidence that the model you built is performing the way you expect. Thinking about machine learning operations, or MLOps, is the process of making your model maintainable and repeatable for a production use case.
+
+### Automating ML model maintenance
+
+Whether you have a large or small model, Dagster can help automate data refreshes and model training based on your business needs.
+
+We can use [auto-materializing assets](/concepts/assets/asset-auto-execution) to update a machine learning model when the upstream data is updated. This can be done by setting the `AutoMaterializePolicy` to `eager`, which means that our machine learning model asset will be refreshed anytime our data asset is updated.
+
+We can also prevent the model from being refreshed too often by using the `max_materilizations_per_minute` setting, which is recommended to have more control over how often the model is retrained. This approach can be suitable for smaller machine learning models where the cost of training is low, and the value of having the complete data used to train a machine learning model is essential.
```

Comment:
`max_materializations_per_minute` is actually already set automatically, to a maximum of 1 materialization per minute.  I think it's unlikely that the upstream data would be updated more than once per minute, so might make sense to leave this out?

Reply:
I think they might want to reduce it further given the context for an ML model

Reply:
freshness policy